client/components/FormattedText/FormattedText.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styles from './FormattedText.module.scss';
+
+type FormattedTextProps = {
+  as?: React.ComponentType | keyof JSX.IntrinsicElements;
+  children?: React.ReactNode;
+} & React.AllHTMLAttributes<HTMLOrSVGElement>;
+
+const FormattedText: React.FC<FormattedTextProps> = ({
+  as: Wrapper = 'div',
```

Comment:
I get a linting error regarding the props:

![image](https://user-images.githubusercontent.com/11327761/125409093-3a87b700-e3c4-11eb-9712-07700edc0944.png)

I think it's enough to do this:

```
const FormattedText = ({
  as: Wrapper = 'div',
  children,
  ...props
}: FormattedTextProps) => {
```

Reply:
ok Let me replace it